app/components/TextEditor/TextEditorConfig.js

```diff
@@ -0,0 +1,13 @@
+const TextEditorConfig = {
+  apiKey: 'fvry0is6hvuc3ij671v897r0t3854qvo6ne0k5eun4378zxk',
```

Comment:
Move apiKey to .env file. Anything that can be different per environment has to go to .env